index.js

```diff
@@ -5,6 +5,7 @@ var constants = require('./lib/constants');
 var assets = require('./lib/loadAssets');
 var errcode = require('err-code');
 var allColors = [];
+var numericSizes = {'s': 11, 'l': 15};
```

Comment:
Not sure I understand why this needs to be accounted for if Maki@7 excludes these size variations?

Reply:
Great question! We still use these sizes when generating the alphabet & numeric SVG symbols to place on top of marker. So this is sticking around purely to account for that variation.